pkg/networkmanager/team.jsx

```diff
@@ -182,7 +182,7 @@
                     </FormSelect>
                 </FormGroup>}
                 <FormGroup fieldId={idPrefix + "-link-watch-select"} label={_("Link watch")}>
-                    <FormSelect id={idPrefix + "-link-watch-select"} onChange={setLinkWatch}
+                    <FormSelect id={idPrefix + "-link-watch-select"} onChange={(_, val) => setLinkWatch(val)}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230503-161849-e27cc6bb-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)